python/lsst/pipe/tasks/snapCombine.py

```diff
@@ -223,18 +223,17 @@ def run(self, snap0, snap1, defects=None):
             table.setMetadata(self.algMetadata)
             detRet = self.detection.run(table, diffExp)
             sources = detRet.sources
-            fpSets = detRet.fpSets
             if self.config.doMeasurement:
                 self.measurement.measure(diffExp, sources)
 
             mask0 = snap0.getMaskedImage().getMask()
             mask1 = snap1.getMaskedImage().getMask()
-            fpSets.positive.setMask(mask0, "DETECTED")
-            fpSets.negative.setMask(mask1, "DETECTED")
+            detRet.positive.setMask(mask0, "DETECTED")
+            detRet.negative.setMask(mask1, "DETECTED")
```

Comment:
ugh "detRet"... Not your fault, and no need to change it here, but it makes me grumpy.